actionlib_params:
    linebreak_debounce_iterations: 10
actionlib_cargo_params:
    roller_power: 1
    intake_timeout: 3 #in seconds
    outtake_timeout: 3 #timeout for the elevator when outtaking
actionlib_hatch_panel_params:
    intake_timeout: 5 #in seconds
    pause_time_between_pistons: 0.1 #time to pause between piston actuations, in seconds
actionlib_lift_params:
    cargo:
        intake_setpoint: 0.0
        cargo_ship_setpoint: 0.5
        rocket1_setpoint: 1.0
        rocket2_setpoint: 2.0
        rocket3_setpoint: 3.0
    hatch:
        intake_setpoint: 0.0
        cargo_ship_setpoint: 0.5
        rocket1_setpoint: 1.0
        rocket2_setpoint: 2.0
        rocket3_setpoint: 3.0
    climber_setpoints:
        deploy_setpoint: -1.0
        climb_setpoint: -1.0
    max_speed: -1.0
    timeout: 4
    elevator_setpoint_deadzone: 1.0
